CMakeLists.txt

```diff
@@ -5,6 +5,9 @@ find_package(Python REQUIRED COMPONENTS Interpreter Development.Module)
 
 python_add_library(deflate_ext MODULE src/deflate.c)
 set_property(TARGET deflate_ext PROPERTY OUTPUT_NAME deflate)
+if(Python_VERSION VERSION_GREATER_EQUAL 3.11)
+  target_compile_definitions(deflate_ext PRIVATE Py_LIMITED_API=0x030b00f0)
```

Comment:
Note to self: I should probably expose this as a SKBUILD variable in the future.

Reply:
Ahh, no, that's not the right way to do this in Modern CMake. Given I helped with the feature, I should have remembered this 🤦. Maybe we could add `SKBUILD_USE_SABI` though to avoid the if statement in the future.

Reply:
And we really need better docs here on Stable ABI.